src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetEvidenceNoteForSchemeRequestHandlerTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿namespace EA.Weee.RequestHandlers.Tests.Unit.AatfEvidence
+{
+    using System;
+    using System.Security;
+    using System.Threading.Tasks;
+    using AutoFixture;
+    using Core.AatfEvidence;
+    using DataAccess.DataAccess;
+    using Domain.AatfReturn;
+    using Domain.Evidence;
+    using Domain.Organisation;
+    using Domain.Scheme;
+    using FakeItEasy;
+    using FluentAssertions;
+    using Prsd.Core.Domain;
+    using Prsd.Core.Mapper;
+    using RequestHandlers.AatfEvidence;
+    using RequestHandlers.AatfReturn.Internal;
+    using RequestHandlers.Security;
+    using Weee.Requests.AatfEvidence;
+    using Weee.Tests.Core;
+    using Xunit;
+
+    public class GetEvidenceNoteForSchemeRequestHandlerTests
+    {
+        private GetEvidenceNoteForSchemeRequestHandler handler;
+        private readonly Fixture fixture;
+        private readonly IWeeeAuthorization weeeAuthorization;
+        private readonly IEvidenceDataAccess evidenceDataAccess;
+        private readonly IMapper mapper;
+        private readonly GetEvidenceNoteRequest request;
+        private readonly Note note;
+        private readonly Guid evidenceNoteId;
+        private readonly Guid recipientId;
+
+        public GetEvidenceNoteForSchemeRequestHandlerTests()
+        {
+            fixture = new Fixture();
+            weeeAuthorization = A.Fake<IWeeeAuthorization>();
+            evidenceDataAccess = A.Fake<IEvidenceDataAccess>();
+            mapper = A.Fake<IMapper>();
+
+            A.Fake<IAatfDataAccess>();
+            A.Fake<IUserContext>();
+
+            A.Fake<Organisation>();
+            A.Fake<Aatf>();
+            A.Fake<Scheme>();
+            note = A.Fake<Note>();
+            fixture.Create<Guid>();
+            evidenceNoteId = fixture.Create<Guid>();
+            recipientId = fixture.Create<Guid>();
+
+            A.CallTo(() => note.Recipient.Id).Returns(recipientId);
+
+            request = new GetEvidenceNoteRequest(evidenceNoteId);
```

Comment:
This will be the new request also

Reply:
line above has been been recoded to:
            request = new GetEvidenceNoteForSchemeRequest(evidenceNoteId);
            handler = new GetEvidenceNoteForSchemeRequestHandler(weeeAuthorization, evidenceDataAccess, mapper);